src/components/Public/Page/Attachment/Uploader.vue

```diff
@@ -95,6 +106,13 @@ export default {
 }
 </script>
 
+<style lang="scss">
```

Comment:
put this to scoped style (if we really need one...) and use /deep/ to override the dependency.